src/main/java/com/example/demo/view/validate/CommandValidator.java

```diff
@@ -0,0 +1,24 @@
+package com.example.demo.view.validate;
+
+import java.util.regex.Pattern;
+
+public class CommandValidator {
+
+    private static final Pattern IS_NUMBER_ONE_TO_THREE = Pattern.compile("^[1-3]{1}$");
+    private static final Pattern IS_NUMBER_ONE_TO_FOUR = Pattern.compile("^[1-4]{1}$");
+
+    private CommandValidator() {
+    }
```

Comment:
유틸성 클래스는 다음과 같이 변경가능성을 막아주세요~

```suggestion
@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
public final class CommandValidator {

    private static final Pattern IS_NUMBER_ONE_TO_THREE = Pattern.compile("^[1-3]{1}$");
    private static final Pattern IS_NUMBER_ONE_TO_FOUR = Pattern.compile("^[1-4]{1}$");

```

Reply:
제가 suggestion 예시에 남긴것처럼 final class로 선언해서 상속을 막으면 메소드 overriding까지 막을 수 있겠죠?

다만 실제로 상속까지 막을 필요는 없습니다. (지금은 최대한 이상적인 방식으로 리뷰를 드리고 있을 뿐) 
apache common langs와 같은 유틸 라이브러리의 코드를 보면 생성자나 상속을 막지 않고 있습니다.